WordPress/Classes/ViewRelated/Blog/Site Settings/Related Posts/RelatedPostsSettingsViewController.m

```diff
@@ -251,7 +251,8 @@ - (IBAction)updateRelatedPostsSettings:(id)sender
     self.settings.relatedPostsShowHeadline = self.relatedPostsShowHeaderCell.on;
     self.settings.relatedPostsShowThumbnails = self.relatedPostsShowThumbnailsCell.on;
     
-    BlogService *blogService = [[BlogService alloc] initWithManagedObjectContext:self.blog.managedObjectContext];
+    BlogService *blogService = nil;
+    // TODO: the save operation in `updateSettingsForBlog` is needed.
```

Comment:
Should this TODO be addressed in the context of this PR?

Reply:
Good catch! I forgot to delete this comment. [It's gone now](https://github.com/wordpress-mobile/WordPress-iOS/pull/20047/commits/c28b72472429cc7a2e552e78ebace85af36be78b).